test_skype_send_message.py

```diff
@@ -1,37 +1,41 @@
 import unittest
 import skype_send_message
+import keyring
+
+
 class skype_send_message_test(unittest.TestCase):
     """skype_send_message tests"""
+
     @classmethod
     def setUpClass(cls):
         """Set up for class"""
-        print ("setUpClass")
-        print ("==========")
+        print("setUpClass")
+        print("==========")
 
     @classmethod
     def tearDownClass(cls):
         """Tear down for class"""
-        print ("==========")
-        print ("tearDownClass")
+        print("==========")
+        print("tearDownClass")
 
     def setUp(self):
         """Set up for test"""
         print("Set up for [" + self.shortDescription() + "]")
-
+        print("password is set ", keyring.set_password("Skype", "nexus12141", "xxxxxxxx"))
```

Comment:
@idumenskyi we need to same servicename(platform) to be able to pass it to the `main`

Reply:
@idumenskyi could you please define a global variable `SKYPE_KEYRING='ims_skype_keyring'` and use it instead of a hardcode?